src/tlog-test-timespec.c

```diff
@@ -98,200 +99,200 @@ main(void)
 
 #define TS(_sec, _nsec) (struct timespec){_sec, _nsec}
 
-#define TEST(_op_list, _a, _b, _exp_res) \
+#define TEST(_op_list, _a, _b, _exp_res, line) \
```

Comment:
Don't add the `line` argument here. Simply use `__LINE__` (and `__FILE__`) in the macro body and they will be turned into the file name and line number of the place where the macro was expanded, by the compiler, and you won't need to pass them every time, below.